src/Symfony/Component/ExpressionLanguage/Tests/ParserTest.php

```diff
@@ -161,4 +161,44 @@ private function createGetAttrNode($node, $item, $type)
     {
         return new Node\GetAttrNode($node, new Node\ConstantNode($item), new Node\ArgumentsNode(), $type);
     }
+
+    /**
+     * @expectedException \Symfony\Component\ExpressionLanguage\SyntaxError
+     */
+    public function testParseWithStringAsInvalidName()
+    {
+        $lexer = new Lexer();
+        $parser = new Parser(array());
+        $parser->parse($lexer->tokenize('foo."#"'), array('foo'));
+    }
```

Comment:
By the way, I found the "fabbot.io" link does not work. When I click on the "Details" link, it says "page not found". Does it matter or is there a way to fix it?

The "travis" build also failed several times, is there anything I can do?